apps/hash-api/src/graph/knowledge/primitive/entity.ts

```diff
@@ -134,9 +139,46 @@ export const createEntity: ImpureGraphFunction<
 export const getEntities: ImpureGraphFunction<
   {
     query: EntityStructuralQuery;
+    temporalClient?: TemporalClient;
   },
   Promise<Subgraph<EntityRootType>>
-> = async ({ graphApi }, { actorId }, { query }) => {
+> = async ({ graphApi }, { actorId }, { query, temporalClient }) => {
```

Comment:
Just want to note in https://github.com/hashintel/hash/pull/3929 the `temporalClient` will become available in the context (i.e. the first parameter here), but since that will likely merge after this PR I'll address this when resolving the conflicts.

Reply:
Thank you